Feature branch

HeyItsGwen opened the pull request:
Opening a pull request to check for stickler errors

HeyItsGwen committed:
Add empty skeleton

added empty skeleton of a bootstrap formatted file structure: index, style, reboot, images folder.

HeyItsGwen committed:
add reboot stylesheet link

forgot it in the template, so added to the project

HeyItsGwen committed:
redid skeleton worked on nav

redid the skeleton having learned more about bootstrap, and started work on the nav

HeyItsGwen committed:
make nav and footer

made the nav and footer, styled and completed the nav, and started on the footer before remembering to commit

HeyItsGwen committed:
add images work on footer

finished the top part of the footer with social media icons and made it responsive using bootstrap and flex, and started work on the bottom part of the footer by adding the cols/sections.

HeyItsGwen committed:
Finish footer

finished the bottom part of the footer and made it responsive

HeyItsGwen committed:
finish middle section, work on left

finished the middle section with top story, culture/travel, and more stories, and filled out the featured stories section.

HeyItsGwen committed:
remove sticky

one word edit to remove non-functional class

HeyItsGwen committed:
add opinion and bottom section

added the opinion section and bottom sections, as well as ads and styling

HeyItsGwen committed:
fix validation errors

fix validation errors brought up by the w3 validator

stickler-ci committed:
Fixing style errors.

HeyItsGwen committed:
add linters

added linters to root

stickler-ci committed:
Fixing style errors.

stickler-ci committed:
Fixing style errors.

HeyItsGwen committed:
edit stickler to ignor reboot

HeyItsGwen committed:
Merge branch 'feature-branch' of https://github.com/HeyItsGwen/Newsweek-Clone into feature-branch

HeyItsGwen committed:
more stickler ignores

woo

HeyItsGwen committed:
fix some stickler errors

fix a few stickler errors

stickler-ci committed:
Fixing style errors.

HeyItsGwen committed:
more stickler stuff

still descending specificity

HeyItsGwen committed:
Merge branch 'feature-branch' of https://github.com/HeyItsGwen/Newsweek-Clone into feature-branch

HeyItsGwen committed:
remove empty line

HeyItsGwen committed:
stickler

HeyItsGwen committed:
stickler stuff

it's all descending specificity from here on out

HeyItsGwen committed:
sticklerrr

can you believe

HeyItsGwen committed:
sticklerss

should be the last of them

HeyItsGwen committed:
do readme

whoops i forgot

HeyItsGwen committed:
fix readme screenshot

HeyItsGwen committed:
make bottom ad responsiveness better

suretrust commented on index.html:
```diff
@@ -0,0 +1,1126 @@
+
+  <title>Newsweek</title>
+
+  <!--Bootstrap CSS-->
+  <link rel='stylesheet' href='./css/bootstrap.min.css' type='text/css'>
```
Using bootstrap's CDN is much better and faster. But not a deal-breaker.

suretrust commented on index.html:
```diff
@@ -0,0 +1,1126 @@
+        Wed, Nov 6, 2019
+      </div>
+      <div id='navLogo' class=-lg-4'>
+        <div class='logoDiv'>'col-5 col
```
There's some error with this line, kindly fix.

suretrust reviewed (changes requested):
Hi @HeyItsGwen,

#### Great job thus far! :raised_hands: Just a few suggestions to wrap this up:

- [x] Please see my comments on your code.

**Large screen**
- [x] The total height of the three images and captions by the right is equal to the height of the ones on the left. Kindly fix as appropriate, refer to the images below.

![height](https://user-images.githubusercontent.com/31304943/68517406-c4928a80-0287-11ea-92e2-abecffc09a08.png)
![height2](https://user-images.githubusercontent.com/31304943/68517412-c9efd500-0287-11ea-9103-b5f1aac38a5b.png)

- [x] The navbar should be sticky. (The sticky navbar is smaller) Please refer to the image below:

![nav](https://user-images.githubusercontent.com/31304943/68517537-44205980-0288-11ea-8ef5-c230196ee541.png)

**Medium screen**

- [x] The navbar has some alignment issues and should be sticky. Please see the image below:

![nav2](https://user-images.githubusercontent.com/31304943/68517625-a1b4a600-0288-11ea-97c8-b4d3222ebe70.png)

**Small screen**
- [x] The navbar has some alignment issues and should be sticky. Please see the image below:

![nav3](https://user-images.githubusercontent.com/31304943/68517717-007a1f80-0289-11ea-8c02-ae4ce68a623b.png)

**Kindly submit for another code review once these issues have been fixed.** _Happy coding! :tada:_

HeyItsGwen committed:
remove weird line

HeyItsGwen committed:
fix genre right images

resize correctly based on reference

stickler-ci committed:
Fixing style errors.

HeyItsGwen committed:
fix nav positioning and alignment

fixed the nav containers alignments, and made sticky act correctly for each breakpoint

HeyItsGwen committed:
stickler errors

HeyItsGwen committed:
Merge branch 'feature-branch' of https://github.com/HeyItsGwen/Newsweek-Clone into feature-branch

stickler-ci committed:
Fixing style errors.

HeyItsGwen committed:
fix validation

w3 validator

HeyItsGwen committed:
last stickler

fix the last stickler error

HeyItsGwen commented:
Thanks, Saheed! Fixed, validated, and resubmitting 😃

Redvanisation reviewed (approved):
# Project approved ✔️ 

### Hello @HeyItsGwen  🌟 

### _Nice work on this project, it seems that you have met the requirements so I am approving your project. 👍_ 

### You may merge to master and submit a project completion form. 🚀 

### Good luck! 🍀

HeyItsGwen commented:
Thank you, Radouane!
